contrib/kind.sh

```diff
@@ -53,9 +53,9 @@ usage() {
     echo "-ho  | --hybrid-enabled            Enable hybrid overlay. DEFAULT: Disabled."
     echo "-ds  | --disable-snat-multiple-gws Disable SNAT for multiple gws. DEFAULT: Disabled."
     echo "-dp  | --disable-pkt-mtu-check     Disable checking packet size greater than MTU. Default: Disabled"
-    echo "-nf  | --netflow-targets           Comma delimited list of ip:port netflow collectors. DEFAULT: Disabled."
-    echo "-sf  | --sflow-targets             Comma delimited list of ip:port sflow collectors. DEFAULT: Disabled."
-    echo "-if  | --ipfix-targets             Comma delimited list of ip:port ipfix collectors. DEFAULT: Disabled."
+    echo "-nf  | --netflow-targets           Comma delimited list of ip:port or :port (using node IP) netflow collectors. DEFAULT: Disabled."
```

Comment:
Instead of leaving this empty to signify the host, why not just have a special enum... Something like `host:8080`, that seems a bit more readable.

Reply:
Note this is kind of a NIT at this point... Otherwise the /lgtm